.github/workflows/ci.yml

```diff
@@ -62,6 +62,8 @@ jobs:
       run: make -s -j4
     - name: make test
       run: make test HARNESS_JOBS=${HARNESS_JOBS:-4}
+    - name: make cmd-nits
+      run: make cmd-nits
```

Comment:
Uhmmmmmmm....  why did you add it to the `basic_gcc` job?
I understand that the `check_docs` job doesn't seem like a good candidate, since it doesn't actually build openssl, but placing it in some other quite arbitrary job doesn't help finding out the type of errors that `cmd-nits` detects.  I would rather have the `check_docs` job actually build openssl, even if that seems annoying.  I believe that `cmd-nits` have the appropriate dependencies for the build to happen automagically.

Reply:
A separate job would also work..

Reply:
You are advocating spending more CI time?  That's a change :confused: 

If a job errors, I look at it.  Finding out the error just happens.  Where this is run don't make much of a difference IMO.

Reply:
The `check_docs` and `check_update` jobs are made separate so that those failures are detected *fast*.  It's also a plus that they are separate, so you can rapidly see that there was such a mistake just by looking at the list of actions.  The purpose is quick dev turnaround time.

You have placed `make cmd-nits` very late, which means that it'll take time to detect that sort of mistake, and it's fairly obscure if you want to look for that sort of mistake specifically (I do)

Reply:
Hmm, I agree with @richsalz (who brought this up) and @paulidale that piggy-backing the `cmd-nits` check with any job that anyway builds the app(s) is most efficient. Yet note that the system executing the Llnux-based CI runs is very fast, so efficiency is not that critical here. 
OTOH I agree with the point by @levitte that it would be nice to be able to see immediately that something is wrong just with the docs, as it was originally in this PR. Yet also note that executing `cmd-nits` may fail also because there was any build issue with the app(s) and the core libs, so they are anyway entangled.

Meanwhile I think that we can live with the minor inefficiency to include building the app with the **check_docs** CI run.
The real solution would be to re-implement `find-doc-nits -c` in a way that does not require building the app(s).

Reply:
> The real solution would be to re-implement `find-doc-nits -c` in a way that does not require building the app(s).

It could be done, actually.  After all, the options are grouped into one array, which isn't impossible to parse...

Reply:
... but that's for a different PR

Reply:
Yes, and in anticipation of that I'd request reverting this PR to its original state, combining `doc-nits` and `cmd-nits.`

Reply:
> The real solution would be to re-implement find-doc-nits -c in a way that does not require building the app(s).

Good luck. :)   There are ifdefs, and macro expansions. It's not trivial.

Reply:
> > The real solution would be to re-implement find-doc-nits -c in a way that does not require building the app(s).
> 
> Good luck. :) There are ifdefs, and macro expansions. It's not trivial.

It took me 3 hours to do it: #15298.

Reply:
@DDvO, did you want to add the cmd-nits test to your PR?  If so, I'll just close this one.

It would just be the first trivial change I made here.

Reply:
Yes, I've added `cmd-nits` to the `doc-nits` CI run there.

Reply:
Great, I'll close this one.